samples/app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "gandi.translate-sample",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "react-scripts": "0.7.0"
```

Comment:
In case of CRA you also need to include `babel-preset-react` (or `babel-preset-react-app`) and the preset to a new `.babelrc` file in order to use `extract_messages` script